### PR TITLE
bot has correct output for migrations

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: conda_forge_tick/auto_tick.xsh [hunk 1]
```diff
@@ -519,15 +519,18 @@ def migration_factory(migrators, gx, pr_limit=50):
...
-        exclude_packages = set(loaded_yaml.get('__migrator', {}).get('exclude', []))
-        bump_number = loaded_yaml.get('__migrator', {}).get('bump_number', 1)
-        package_names = ((set(loaded_yaml)|set(l.replace('_', '-') for l in loaded_yaml)) & set(gx.nodes)) - exclude_packages
         print(os.path.splitext(yaml_file)[0])
-        add_rebuild_migration_yaml(migrators, gx, package_names, yaml_contents,
-                                   migration_name=os.path.splitext(yaml_file)[0],
-                                   pr_limit=pr_limit,
-                                   obj_version=obj_version, bump_number=bump_number)
+
+        yaml_config = loaded_yaml.get('__migrator', {})
```
Comment (mariusvniekerk) [inline conda_forge_tick/auto_tick.xsh]: naming nit
```suggestion
        migrator_config = loaded_yaml.get('__migrator', {})
```

File: conda_forge_tick/auto_tick.xsh [hunk 2]
```diff
@@ -519,15 +519,18 @@ def migration_factory(migrators, gx, pr_limit=50):
...
+        exclude_packages = set(yaml_config.get('exclude', []))
+        package_names = ((set(loaded_yaml) | set(l.replace('_', '-') for l in loaded_yaml)) & set(
+            gx.nodes)) - exclude_packages
+
+        add_rebuild_migration_yaml(
+            migrators, gx, package_names, migration_yaml=loaded_yaml,
+            migration_name=os.path.splitext(yaml_file)[0], config=yaml_config,
```
Comment (mariusvniekerk) [inline conda_forge_tick/auto_tick.xsh]:
```suggestion
            migration_name=os.path.splitext(yaml_file)[0], config=migrator_config,
```

**Comment by CJ-Wright** [comment]
Merging. I'll babysit the bot for a bit and then move forward with our migration backlog.

File: conda_forge_tick/migrators.xsh [hunk 3]
```diff
@@ -1322,16 +1321,18 @@ class MigrationYaml(Migrator):
     """Migrator for bumping the build number."""
     migrator_version = 0
     rerender = True
-    bump_number = 0
```
Comment (ocefpaf) [inline conda_forge_tick/migrators.xsh]: This means: default is to bump unless declared, right? If so bumping is a True/False or an acutal value to bump? Like,

```
  build_number:
    1
```

makes me think that is +1 to the build number, right?

Comment (CJ-Wright) [reply conda_forge_tick/migrators.xsh]: yes, `1` means +1

**Comment by CJ-Wright** [comment]
Lets give this a whirl

**Pull request merged**